src/api/Feed.js

```diff
@@ -663,9 +664,10 @@ class Feed extends Base {
             -use result of filter to process each group sequentially */
             const taskAssignments: Array<TaskCollabAssignee> = flatten<TaskCollabAssignee, TaskCollabAssignee>(
                 await Promise.all(
-                    assignees.map((assignee: SelectorItem): Promise<Array<TaskCollabAssignee> | TaskCollabAssignee> =>
-                        // $FlowFixMe
-                        assignee.type === 'group'
+                    assignees.map((assignee: SelectorItem<UserMini | GroupMini>): Promise<
+                        Array<TaskCollabAssignee> | TaskCollabAssignee,
```

Comment:
The logic is getting difficult to read due to the line length and nesting. Would adding some named methods help?

Reply:
We will be replacing this code soon with a transactional API call, instead of these nested `Promise.all`s.